otx/algorithms/classification/tasks/inference.py

```diff
@@ -346,7 +346,6 @@ def _init_recipe(self):
         # FIXME[Soobee] : if train type is not in cfg, it raises an error in default INCREMENTAL mode.
         # During semi-implementation, this line should be fixed to -> self._recipe_cfg.train_type = train_type
         self._recipe_cfg.train_type = train_type.name
-        patch_data_pipeline(self._recipe_cfg, self.data_pipeline_path)
```

Comment:
@supersoob I'm not sure which PR removed this line but could you check if this is OK?